src/common/meta/src/key/flow/table_flow.rs

```diff
@@ -188,7 +190,9 @@ impl TableFlowManager {
     }
 
     /// Retrieves all [TableFlowKey]s of the specified `table_id`.
-    pub fn nodes(&self, table_id: TableId) -> BoxStream<'static, Result<TableFlowKey>> {
+    ///
+    /// TODO(discord9): add cache for it since range request does not support cache.
```

Comment:
Seems we can cache all metadata of flow, like what we have done with table metadata. cc @WenyXu

Reply:
Oh, now we can only cache it on the metasrv leader. It seems that the frontend needs this cache now.